docs/en/latest/plugins/prometheus.md

```diff
@@ -93,6 +93,20 @@ Then add prometheus plugin:
 
 We fetch the metric data from the specified url `/apisix/prometheus/metrics`.
 
+First you need to setup the route for the metrics API, which will use the [public-api](public-api.md) plugin.
+
+```shell
+$ curl http://127.0.0.1:9080/apisix/admin/routes/p -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+{
+    "uri": "/apisix/prometheus/metrics",
+    "plugins": {
+        "public-api": {}
+    }
+}'
+```
+
+Then, we can get the metrics.
```

Comment:
We don't need to do it for prometheus, which uses a different port.

Reply:
I think now that we have introduced `public-api` plugin, can we reduce the existence of APIs with special configuration items on these plugins and let users give priority to using public-api plugin to realize similar functions, such as `prometheus` plugin and `batch-requests` plugins.

Reply:
The default port of Prometheus metrics is 9091, not the 9080.

Reply:
It is not recommended to configure the port to 9080, which abuses the 9080 port and should be only used in the CI.

Reply:
I think we should remove the example of public-api in prometheus. Better not encourage the abuse.

Reply:
changes rolled back